go/logic/inspect.go

```diff
@@ -195,12 +196,22 @@ func (this *Inspector) validateConnection() error {
 	if len(this.connectionConfig.Password) > mysql.MaxReplicationPasswordLength {
 		return fmt.Errorf("MySQL replication length limited to 32 characters. See https://dev.mysql.com/doc/refman/5.7/en/assigning-passwords.html")
 	}
-	query := `select @@global.port, @@global.version`
+
+	extraPortRegexp := regexp.MustCompile("Percona|Maria")
```

Comment:
I just realized yesterday that Enterprise MySQL offers a thread pool, and presumably an extra port as well. I don't have access to a running version of that though (only Community). But this could be extended to include that as well if someone does have access or knows a valid way to differentiate.